lib/support/video/scripts/pullNetLogAndroid.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+
+const Promise = require('bluebird');
+const fs = require('fs');
+
+module.exports = {
+    run(context) {
+        const taskData = context.taskData;
+        context.log.debug('Pull netlog');
+        const file = `/sdcard/chromeNetlog-${ context.index}.json`
```

Comment:
yes we do that when we set the chrome options, check https://github.com/sitespeedio/browsertime/pull/332/files#diff-1c3ad552ac7bae72e3768fea0e5bb02eR108